### PR TITLE
Create desk top version

### PR DESCRIPTION
**Pull request by wayungi**
### This PR adds  the following
- CSS styles for screen sizes larger than 759 pixels
- Image scaling as page size increases
- New background images for larger screen size

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,239 +9,200 @@
...
+              sign-ups required.
+            </p>
+            <div class="language">
+              <a href="#" class="font">html</a>
+              <a href="#" class="font">css</a>
+              <a href="#" class="font">javascript</a>
+            </div>
+            <button type="submit" class="font">See Project</button>
+          </article>
+        </div>
```
Comment (TedLivist) [inline index.html:131]: - [ ] Kindly add the appropriate margin between these two works as contained in the Figma design.

  wayungi-space

File: index.html [hunk 2]
```diff
@@ -9,239 +9,200 @@
...
+             a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form action="https://formspree.io/f/xqknlvdq" method="post">      
+            <input type="text" class="font" id="name" name="user_name" placeholder="Type your name here" maxlength="30" required>          
+            <input type="email" class="font" id="email" name="user_email" placeholder="Type your email here" required>          
+            <textarea id="message" class="font" name="user_message" placeholder="Write your message here" maxlength="500" required></textarea>
+            <button type="submit">Get in touch</button>
+          </form>
+          <div id="contact-page-scroll"></div>
+        </section>
       </div>
```
Comment (TedLivist) [inline index.html:205]: - [ ] Your **About Myself** and **Contact** sections are missing the top-right and top-left border radius attributes as we have in the Figma template. Kindly add them.

  Your Implementation | Figma Design
  :---:|:---:
  wayungi-footer | wayungi-figma-footer

File: index.html [hunk 3]
```diff
@@ -9,239 +9,200 @@
...
+              <a href="#" class="font">javascript</a>
+            </div>
+            <button type="submit">See Project</button>
+          </article>
+        </div>
+
+        <div class="card-2 card">
+          <div class="card-partition re-order">
+            <img src="images/portfolio2.svg" alt="portfolio image" />
+          </div>
```
Comment (TedLivist) [inline index.html:88]: - [ ] I would recommend you stick to the design as much as possible. Kindly ensure that there is somewhat proportional spacing on the left and right sides of the works as we have in the template

- [ ] Your card 1 and card 3 have some issues. As I maximize my screen size, there top and bottom spaces of the work images seem to disappear. Kindly fix this.

- [ ] Your text on the work cards are somewhat aligned to be vertically centered but in the Figma template they are positioned on the top. Kindly fix this.

- [ ] Kindly make your work titles to be larger as we have in the template. This makes it more legible as a title.

- [ ] **[OPTIONAL]** For a better user experience I would recommend you set the **cursor** attribute to `pointer` for your buttons, like so: `cursor: pointer`.

  Your Implementation | Figma Design
  :---:|:---:
  wayungi-html-spaces | wayungi-figma-spaces

File: README.md [hunk 4]
```diff
@@ -30,12 +30,6 @@ To get a local copy up and running follow these simple example steps.
 - Twitter: [@twitterhandle](https://twitter.com/FrancisWayungi)
```
Comment (TedLivist) [inline README.md:30]: - [ ] In your README, kindly add detailed instructions on how users can clone your repo.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @wayungi,

Good job so far!
✔️ Nice design
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by akucintavalent — APPROVED** [review]
# Approved 👏 

Hi there, @wayungi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error at present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Optional suggestions:
- [ ] [OPTIONAL] Kindly fix the distances shown on the picture below

Your design:
Zrzut ekranu 2022-03-25 o 23 00 01

Figma template:
Zrzut ekranu 2022-03-25 o 23 01 08

- [ ] [OPTIONAL] Kindly fix the distances shown on the picture.
Your design:
Zrzut ekranu 2022-03-25 o 23 04 26

Figma design:
Zrzut ekranu 2022-03-25 o 23 02 42



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**